chapters/playframework/extra/code/metrics/README.md

```diff
@@ -0,0 +1,34 @@
+# How to view these metrics
+
+Copy [this json file](https://github.com/delftswa2014/team-playframework/releases/download/metric-modules-output/metricscache.json) 
```

Comment:
Damn, you did a regex search. I can only fix this after the weekend however.

@michielha?